bootloader/wscript

```diff
@@ -661,6 +661,9 @@ def build(ctx):
     if ctx.env.CC_NAME == 'msvc':
         # Do not strip bootloaders when using MSVC.
         features = ''
+        
+        ctx.env['WINRC_TGT_F'] = '/fo'
+        ctx.env['WINRC_SRC_F'] = ''
```

Comment:
I don't think it is unrelated because [it does compile][1] when those variables are set in configure.

[1]: https://ci.appveyor.com/project/xoviat/pyinstaller/build/1.0.212/job/wttbv6tb5een3wir